### ISSUE TITLE
15:30 June 9 - rm319 Biochem. R Functions

### ISSUE THREAD

**Issue by murraycadzow** [OtagoStudyGroup/studyGroup#32]
This weeks study session is going to be on R functions

Based on [software carpentry lesson 2](http://swcarpentry.github.io/r-novice-inflammation/02-func-R.html). We'll be covering making functions in R. Future lessons will depend heavily on what is covered here, especially when we eventually cover making packages and testing code.

Hopefully there will be some extra tricks for you to learn to make things easier

come along (with a laptop running R/RStudio for maximum benefit)

**Comment by TomKellyGenetics** [comment]
For those that attended last week, we touched on this but our focus was on using functions and understanding their arguments. Murray will go into more detail into how to write your own. This is a handy trick that could save you a lot of repetitive work so I'd still recommend going.

This is a topic that we've often had to brush over in our full-day workshops (in the interest of time) so it would be suitable for anyone who found those workshops too fast / overloaded. See? I did read the feedback :)
